src/data/finset/pointwise.lean

```diff
@@ -323,6 +333,13 @@ coe_injective.mul_one_class _ (coe_singleton 1) coe_mul
 localized "attribute [instance] finset.semigroup finset.add_semigroup finset.comm_semigroup
   finset.add_comm_semigroup finset.mul_one_class finset.add_zero_class" in pointwise
 
+@[to_additive] lemma subset_mul_left (s : finset α) {t : finset α} (ht : (1 : α) ∈ t) : s ⊆ s * t :=
+λ a ha, mem_mul.2 ⟨a, 1, ha, ht, mul_one _⟩
+
+@[to_additive] lemma subset_mul_right {s : finset α} (t : finset α) (hs : (1 : α) ∈ s) :
+  t ⊆ s * t :=
+λ a ha, mem_mul.2 ⟨1, a, hs, ha, one_mul _⟩
```

Comment:
It's tempting to add `_of_one_mem` to these names to match the pow and univ lemmas below

Reply:
The pow and univ lemmas are named like this because the `_of_one_mem`-less version already exists. Here, given the statement, `1 ∈ s` is the only sensible thing to ask for. And `set.subset_mul_left`/`set.subset_mul_right` already exist.

Reply:
> And `set.subset_mul_left`/`set.subset_mul_right` already exist.

Out of scope for this PR, but; Can you think of any ways to make this easier to keep track of? Maybe a `@[see_also set.subset_mul_right]` which checks the lemma exists? It's a little more tricky if the similar lemmas are in two files that don't import each other.

Reply:
In this specific case, it would probably bze solved if we had a `to_additive`-like attribute to turn set statements into finset ones.

Reply:
> And `set.subset_mul_left`/`set.subset_mul_right` already exist.

Good enough for me